aws-cs-eks/EksStack.cs

```diff
@@ -54,10 +54,11 @@ class EksStack : Stack
     public EksStack()
     {
         // Read back the default VPC and public subnets, which we will use.
-        var vpc = Output.Create(Ec2.GetVpc.InvokeAsync(new Ec2.GetVpcArgs {Default = true}));
-        var vpcId = vpc.Apply(vpc => vpc.Id);
-        var subnet = vpcId.Apply(id => Ec2.GetSubnetIds.InvokeAsync(new Ec2.GetSubnetIdsArgs {VpcId = id}));
-        var subnetIds = subnet.Apply(s => s.Ids);
+        var vpcId = Ec2.GetVpc.Invoke(new Ec2.GetVpcInvokeArgs { Default = true })
```

Comment:
What's bad about that?

Reply:
Well in general Output is more constrained in what you can do with it than Task. I don't think it really matters here because the inputs are always going to be known and so these invokes will run anyway just wondering out loud if this is going to make things more difficult with preview constraints as even more stuff is in Outputs.

Reply:
We had a similar debate in TypeScript with Pat eventually resolved it to the rule "if the change makes Output appear that was not there before, it's a bad change". I'm happy to roll it back.

Reply:
Invokes should not be running in preview AFAIK.

Reply:
Invokes should run in preview if the Inputs are known though?

Reply:
Old-style invokes totally run in preview unless inside an unresolved apply